src/screens/title.rs

```diff
@@ -3,54 +3,38 @@
 use bevy::prelude::*;
 
 use super::Screen;
-use crate::ui::prelude::*;
+use crate::ui::{interaction::OnPress, prelude::*};
 
 pub(super) fn plugin(app: &mut App) {
     app.add_systems(OnEnter(Screen::Title), show_title_screen);
-
-    app.register_type::<TitleAction>();
-    app.add_systems(Update, handle_title_action.run_if(in_state(Screen::Title)));
-}
-
-#[derive(Component, Debug, Clone, Copy, PartialEq, Eq, Reflect)]
-#[reflect(Component)]
-enum TitleAction {
-    Play,
-    Credits,
-    /// Exit doesn't work well with embedded applications.
-    #[cfg(not(target_family = "wasm"))]
-    Exit,
 }
 
 fn show_title_screen(mut commands: Commands) {
+    let on_play = commands.register_one_shot_system(on_play);
+    let on_credits = commands.register_one_shot_system(on_credits);
+    let on_exit = commands.register_one_shot_system(on_exit);
```

Comment:
```suggestion
    #[cfg(not(target_family = "wasm"))]
    let on_exit = commands.register_one_shot_system(on_exit);
```
To avoid a clippy warning.